pkgs/os-specific/linux/systemd/0019-Revert-pkg-config-prefix-is-not-really-configurable-.patch

```diff
@@ -1,4 +1,4 @@
-From cd5b1075499b8498d9c700a317ad11a3199c447a Mon Sep 17 00:00:00 2001
+From ba59e81758c3ae0382ee491181869ae4338a0f22 Mon Sep 17 00:00:00 2001
 From: =?UTF-8?q?J=C3=B6rg=20Thalheim?= <joerg@thalheim.io>
 Date: Sun, 6 Dec 2020 08:34:19 +0100
 Subject: [PATCH 19/19] Revert "pkg-config: prefix is not really configurable,
```

Comment:
This is not a revert of that commit anymore, the commit message doesn't apply anymore.
Upstream reverted most of the commit we initially reverted in this patch, except the
hardcoding of `prefix=/usr`.

We should change the commit message to something like

```
pkg-config: derive prefix from --prefix

Point prefix to the one configured, instead of `/usr` `systemd` has limited
support for making the pkgconfig prefix overridable, and interpolates those
values later down.

So we only need to patch this one value to get the correct paths.
See systemd/systemd@bc4e6e27922a2873985ab9367d79fb099f70b505 for details.
```

Reply:
Can you force push this?

Reply:
sure!